src/runtime/styled.js

```diff
@@ -10,16 +11,16 @@ const camelCase = str =>
   );
 
 function styled(type, options, settings) {
-  const { displayName, attrs, styles, kebabName, camelName } = settings;
+  const { displayName, attrs, styles } = settings;
 
   options = options || { allowAs: typeof type === 'string' };
-  const componentClassName = styles[kebabName] || styles[camelName];
+  const componentClassName = styles.cls2 || styles.cls1;
```

Comment:
no real reason to not used a static class name here and avoid some extra work

Reply:
oh, well that is cool.

i don't suppose it'd make sense to mangle the name a bit more? seems unlikely that anyone will use `cls1` as a class name, though

Reply:
maybe we should use an emoji?